docs/pages/database-access/guides/redshift-serverless.mdx

```diff
@@ -0,0 +1,400 @@
+            "Action": [
+                "iam:AttachRolePolicy",
+                "iam:PutRolePermissionsBoundary"
+            ],
+            "Resource": [
+                "arn:aws:iam::(=aws.aws_access_key=):role/teleport-redshift-node"
+            ],
+            "Effect": "Allow"
+        },
+        {
+            "Action": [
+                "iam:GetPolicy",
+                "iam:CreatePolicy",
+                "iam:TagPolicy",
+                "iam:ListPolicyVersions",
+                "iam:CreatePolicyVersion"
+            ],
+            "Resource": [
+                "*"
+            ],
+            "Effect": "Allow"
+        }
+    ]
+}
+```
+
+Remember to replace the example AWS account ID.
+
+<Admonition type="tip">
+This policy is required for Automatic Teleport bootstrapping, documented below.
+You can optionally skip this policy, and select the "Manual" option when you
+reach step 4.
+</Admonition>
+
+Apply this role to the EC2 instance the Teleport Database Service will be
+installed in.
+
+## Step 2/5. Create an IAM Role for user access
+
+Create an AWS IAM role to provide user access to Redshift Serverless. This role
+will be granted to Teleport users via a corresponding Teleport role. In this guide
+we will use the example name `teleport-redshift-serverless-access`.
+
+Under **Trusted entity type** choose "Custom trust policy". Edit the trust policy
+to allow the IAM role generated in the previous step to assume this role:
+
+```json
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Sid": "Statement1",
+            "Effect": "Allow",
+            "Principal": {
+                "AWS": "arn:aws:iam::(=aws.aws_access_key=):role/teleport-redshift-node",
+                "Service": "ec2.amazonaws.com"
+            },
+            "Action": "sts:AssumeRole"
+        }
+    ]
+}
+```
+
+Remember to replace the example AWS account ID.
+
+Create and apply a permission policy to allow access to Redshift Serverless.
+This policy can allow access to all instances:
+
+```json
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Action": "redshift-serverless:GetCredentials",
+            "Resource": "*"
+        }
+    ]
+}
+```
+
+Or you can restrict the `Resource` line to a specific Redshift Serverless workgroup:
+
+```json
+{
+...
+      "Resource": "arn:aws:redshift-serverless:us-west-2:1234567890:workgroup/some-workgroup-id"
+...
+}
+```
+
+## Step 3/5. Create a Teleport role for Redshift Serverless access
+
+On your workstation logged in to your Teleport cluster with `tsh`, define a new
+role to provide access to Redshift Serverless. Our example file is
+`redshift-role.yaml`:
+
+```yaml
+version: v5
+kind: role
+metadata:
+  name: redshift-serverless-access
+spec:
+  allow:
+    db_labels:
+      '*': '*'
+    db_names:
+    - dev
+    db_users:
+    - 'teleport-redshift-serverless-access' # only the role name, do not specify the full AWS ARN
```

Comment:
hmm.. this sounds more like a product bug. 

@smallinsky what do you think on this issue? Do we want to keep it "dumb" as it is (where `--db-user` must be one of `db_users`? Or should we allow mix-match between user name and ARNs? Or maybe force to specify ARN `--db-users`?

Reply:
@smallinsky created #20359 to track this

Reply:
Sounds like we don't need to do any action or consider this a blocker, then?

Reply:
I'm going to assume "yes" to that question.